backend/routes/model.js

```diff
@@ -1939,6 +2043,90 @@ function downloadLatest(req, res, next) {
 	});
 }
 
+async function uploadModelRequest(req, res, next) {
+	const responsePlace = utils.APIInfo(req);
+	const account = req.params.account;
+	const model = req.params.model;
+	const username = req.session.user.username;
+
+	try {
+		const corID = await Upload.uploadRequest(account, model, username, req.body);
+		responseCodes.respond(responsePlace, req, res, next, responseCodes.OK, corID);
+	} catch(err) {
+		const errMsg = err.resCode ? err.resCode : err;
+		responseCodes.respond(responsePlace, req, res, next, errMsg, errMsg);
+	}
+}
+
+async function initUploadChunks(req, res, next) {
+	const responsePlace = utils.APIInfo(req);
+	const account = req.params.account;
+	const model = req.params.model;
+	const corID = req.params.corID;
+	const user = req.session.user.username;
+
+	try {
+		const initHeader = await Upload.initUploadChunks(account, model, corID, user, req.headers);
+		initHeader.Location = `${config.public_protocol}://${req.headers.host}${req.originalUrl}`;
+		res.writeHead(200, initHeader);
+		res.end();
+	} catch(err) {
+		const errMsg = err.resCode ? err.resCode : err;
+		responseCodes.respond(responsePlace, req, res, next, errMsg, errMsg);
+	}
+}
+
+async function uploadChunk(req, res, next) {
+	const responsePlace = utils.APIInfo(req);
+	const account = req.params.account;
```

Comment:
use deconstructed assignment ?
```suggestion
	const { account, model, corId}  = req.params;
```